src/cards/HaAreaCard.js

```diff
@@ -0,0 +1,35 @@
+import {AbstractCard} from "./AbstractCard";
+
+/**
+ * HA Area Card Class
+ *
+ * Used to create a card for an entity of the area domain using the built in type 'area'.
+ *
+ * @class
+ * @extends AbstractCard
+ */
+class HaAreaCard extends AbstractCard {
```

Comment:
Since the file is another module, you can give the class the same name `AreaCard` to avoid an extra check for which class to instantiate.

Reply:
Ok!